build.gradle

```diff
@@ -14,18 +14,19 @@ configurations {
 
 repositories {
   jcenter()
+  google()
 }
 
 dependencies {
-  compile 'org.codehaus.groovy:groovy-all:2.4.4'
+  compile 'org.codehaus.groovy:groovy-all:2.4.12'
```

Comment:
Gradle error message (from a different project, but should be the same here):
````
Configuration 'compile' is obsolete and has been replaced with 'implementation'.
It will be removed at the end of 2018
````

Reply:
Good point! I amended my commit. Thanks.